myStyle.css

```diff
@@ -0,0 +1,38 @@
LCHCBzT4z28z5lNqbACDr28bIPEvPMYyAOniihJbWzJvaftt1j1XfAdMBpJ6BM8WxsANkmLkgCNr7X1Qbnuy5nW6ruTI/HEQtbBRThaUI5EUH6FOjzZ32F4eqE1Dg4ArPuOydcPYTTXNiwTTCKYKlYyw5HLoVXhcTEg5pgykxwkZoMvBWtA2DLWktcPt4hQrAmRrmoVAZgiFISRIQO9UVsAcCSCHCziNRoTv3VLNiL3v5JgaZj5TB0OY7OVL2NdbJ0ZG/iDqE0WSyY32KOIUXNMtvuIkEakffUJVW4Zzw6nrcsm19Wnb3Zakkc4kZSO0Wke9VRTw7C6AOVxkx/S7ct2cpTx3tGrD5Naf+/ILw/hlNrYeCCbhzbntGttLqbeABrhUpvMHJzD8nb/aen0TfB4cklp2yOY97qNXC1KckXDrSNY/1Rr1sUUgNu20w3D4CpyMLuWGmRym/pG+6Zu5SCDJkQQ6DO6QYPHcmpGpBBI8c/RPaGOw9SPiACbCTNPoReAlm5WasLxyW+zHYdhw2I5mO+UmDndhP1FvJbPD8Wc0ybg9/RLcT+lHvcXGsACZ+Vuh2utDg+Htp0w03AES43PeVDx04pqS16NvlrnKLg91sLo49rhmFBwa7QEZ3vffzEpOHsLyGUqjCDBnla13/AB5nQ4dii6NUwTcRnMW16gjqJFlal6Ivmv3oNNFszyyS3lkzlmBMyBOyAp4FsQ9tzIBb/TnFxrrJ31VlPE5EGRuLgjOykyujTRNuLM9i+CEPMNc5sFxNwCJ6ZEXtrAQNVjRDSC5xt4ZXjIjx8FtP8xoTbWEg4rh2srSB+4Rn8t7+EjKNeqtjlemY82FR+qJn69Xci98hYaR0yy6IUV9NcoIjsB5phV5QLt75m51vkPygWUWky2SJvbfaNFUyNtMJpWPM03GcTPW+SOw2Nc3WOlvX2M0ur03iHUzEZAxYWnKL6r2jWESYJ2se+SVotGbQ/bjzHOD63tcIvC4gcwdue3WVlv8APR22t5IvAY24BAI7230SuBphn2jf4Am9tZneyMbU37HaUi4DjBy8s3BOunrlKcOqSsc40z1scrjZa5yoqusu5lVUMoJDNi+vJPe3vdVNbGviFLF55/x9l7RpkjlynPPIXvHgrrozPbJtZvdRdRk5R7CqcYN9M/RGYWoJug9BVPRQ7CnYpVjaIJtlt+VrQARCU8aaAByjLMjP30Qjk2HJiVCRnCw7PaBlGiW8Q4WWuJ5TAHMYyAyP5RFfHubkbbmPvZB4rjVWOV1S0ZNLeW9tLFM5MlxjRlWK2ELh3osBUR401TPWMunfDa0CEmY5G0BqmpUS5NO0G4yhqFbwzEhpjJU0sVFiiWUQWyM0JIrjau0MMeA5nMACR9EsJBA+33XlDGcpLcir2OE8wA6hIjRJqQM6pynODptZRq0HF7ajNp8dbJu/BtcJIs7P3oltGr8Jxp1J5f6THkUUxJQrsi5xdZ7Ym3N1z8tl6+lABdB0MWMjJw6pnVZaw5mlB16Q1EtOfbdMmI8dHtHEwOYHmHv/ALXeiascHftOecj0cB3WcqN+AZaOZjjc69jvbzRNPFlg5mAuZpy/uAOw1b000hBoMZOOmMK+A5iRlfI5eB07IOpwlwJ5T3H5H3TjDVm1Gt/0mYM+l8tiCoip8IQ79uhuYnef2pGrNEWo7FLxiaTB8Gs5sZtty+AIMdgk9T9XY9pg1AehYyf/AB/C2T6jXaCT6/8AE6rP8Z4e10mPE6d9keKYHknDaehZT/W1c2dVLTtysH0amOE4uX/veX/8jI9clnsVwwgXbI6IB/DnsvTJ6BFXH0I5Kbvk1+Xo+nU8YIVjcSF8xo8YrUzDibeITnC8ekDXsQgmmVblFbN3z2lDVYOaQs462wJRdHGsdk7PK6ZAlOynGsHMSRJP9Um2WQFpz81W3AAFr5vcgCcxaTeJPvJMvgtMHXrl5JdUY9hn9wG59FVUzNJOLtnVcQHDLMGcifLLxS/E0InlyGc53ur8TW5rxBm5B16Qq6uKOufl79+BEbvsBNY3n6+/NXYGrfXpt7ugviAOIuVbRqOzE2va/T7oDI1/Ca5FVvUwR4dctFrC8D3qsRgaJD2ua/Nws4AwQ2TPh71WldWgeOYMz7y8FnyRtnreNNqLsYCqpTKWGvLSRf62S0fqENcGv++duilxNPNGgq0cyFxIaI17e90qP6opZFj/ABA8dV6f1FSMQ1xB7flC2Nr0GPgjLXRKeIcTZRydzHO2cTHpKsr8SDhax7x6gpLi8G+oOYuaSNQBH0JHbKyZMnIKP6m/5XtAE+s/RBYjitS/IX8uoi2fn/ZWUsC63zWBJsIv1yHovcQCG8vOc5PzO/KFg2Karn1PmJdOsAm2XYBU1MMZgz42+qOLRnzX3i6qMA/KRf3qutgpGWpOTGi+y5crwPHzI5xuiKVcheLkxBlwfqm+Br6LlyZ9AxupAXE6fzSLI/CkFoMLxcpezY/2l+Hx5Dg03aUTxHCh8D+rQ9NiuXIvTBF3F2V4R5b8hi2RHREOF+h+q5cgMugDH4YH5TqdEvp0DRbMmCYicnHUbg7FcuTIlJdhXC8UZJbaT840Ltxt3TjD1/ifI7MzBHS0HouXIyOxN6QBUpQ2pABa27mOu06y3YquliTygm7I1/e3oDqO65citiyfF0v9shUwoJIFvfl6KivQaBcdJFj4rlyYnJIArYFrr2v0SfF8Pa07GcwuXJZJNDYpyUqTBDTe2YfPdTpcVe3w2Xi5QeujfD6ux1hP1A4wCM4/KY//ACc5g/3XLlSDbI5Vx6KH4iYgIOu4nxy9+a5cqmZgtQyVcx3IDrt4FcuXDfYf/p2o2HOIPPMSNrW7XTjFVgGWlcuSPs3Yn9ADh+JOFlKtiif7BcuS5FsfDJtbKnV3Gxgq9tXQ/RcuUmjQmTMnIwPVcwuGRXq5LQ1knyQZQNay5cuCUYiOUboQlcuXMCP/2Q==") no-repeat fixed center; 
```

Comment:
I typically see something like this:
`background-image: url("paper.gif");` (https://www.w3schools.com/cssref/pr_background-image.asp)
Is that long string the data of the actual image?  If so, then it is more common to save the file as a file and then link to it by filename.
So for example, save the image as `background.jpg` and then reference it like:
`background-image: url("background.jpg");`